src/main/kotlin/com/ampnet/identityservice/service/UserService.kt

```diff
@@ -6,4 +6,5 @@ interface UserService {
     fun getUser(address: String): User
     fun connectUserInfo(userAddress: String, sessionId: String): User
     fun updateEmail(email: String, address: String): User
+    fun createUser(address: String)
```

Comment:
The function should return created user.

Reply:
Why, what to do with the returned value?